### PR TITLE
Sunu's branch

### PR DESCRIPTION
**Pull request by sunu09**
Branch has merge complex due to the re-design help.html.

### CODE REVIEW

**Review by vibrio81 — APPROVED** [review]
reviewed. gradle.bat red but no differences...?

**Review by vibrio81 — APPROVED** [review]
changes ok

**Review by randyhr318 — APPROVED** [review]

File: saveoursons/src/main/java/com/tts/saveoursons/Runner.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    @Override
+    public void run(String... args) throws Exception {
+
+    }
+
+   // @Override
+//    public void run(String... args) throws Exception {
+//        crystalRepository.save(new Crystal("clear", "clear quartz"));
+//        crystalRepository.save(new Crystal("purple", "amethyst"));
+//        crystalRepository.save(new Crystal("yello", "citrine"));
```
Comment (vibrio81) [inline saveoursons/src/main/java/com/tts/saveoursons/Runner.java]: typo-missing "w" in yellow

File: saveoursons/src/main/resources/templates/ReportNewCase.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<!-- TODO(): Update spring security to version 5 -->
+<html xmlns:th="http://www.thymeleaf.org" xmlns:sec="http://www.thymeleaf.org/thymeleaf-extras-springsecurity4">
+
+<head>
+
+
+
+    <!--    <div th:fragment="head-styles">-->
+    <div th:replace="fragments/stylesFragment :: head-styles"></div>
```
Comment (vibrio81) [inline saveoursons/src/main/resources/templates/ReportNewCase.html]: wrong fragment path "fragments/fragments :: head

File: saveoursons/src/main/resources/templates/ReportNewCase.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+
+    <!--    <div th:fragment="head-styles">-->
+    <div th:replace="fragments/stylesFragment :: head-styles"></div>
+
+</head>
+
+<body>
+<div class="container-fluid">
+    <div th:replace="fragments/navbar"></div>
```
Comment (vibrio81) [inline saveoursons/src/main/resources/templates/ReportNewCase.html]: wrong fragment path "fragments/navbar :: navbar"

**Review by vibrio81 — CHANGES_REQUESTED** [review]
a few code typos commented in files

File: saveoursons/src/main/java/com/tts/saveoursons/Runner.java (hunk 1 above)
Comment (vibrio81) [reply saveoursons/src/main/java/com/tts/saveoursons/Runner.java]: missing w on yellow line 25

**Pull request closed without merging**